routes/web.php

```diff
@@ -13,6 +15,8 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/home', [HomeController::class, 'index']);
+
+Route::resource('/article', ArticleController::class);
```

Comment:
```suggestion
Route::resource('articles', ArticleController::class);
```
sửa những chỗ liên quan và tương tự nha